### PR TITLE
 Twitter_Bot: Relevant tweet only be scraped

### PR DESCRIPTION
**Pull request by ananthanandanan**
- Only those tweets with relevant keywords will get stored in tid_store rest will be stored in a backup file.
- Added testCases for the twitter_bot module

### CODE REVIEW

File: twitter_bot.py [hunk 1]
```diff
@@ -39,12 +44,19 @@ def nawab_read_list(self):
...
+            dicts = { 'Date_time' : [time.strftime("%m/%d/%Y %I:%M:%S %p ")],
+                    'Id': [str(tweet_id)]}
+            data = pd.DataFrame(dicts)
+            data.to_csv(self.dirpath + 'tid_store.csv',mode='a',header=False,index=False)
+        else:
+            dicts = { 'Date_time' : [time.strftime("%m/%d/%Y %I:%M:%S %p ")],
+                    'Id': [str(tweet_id)]}
+            data = pd.DataFrame(dicts)
+            data.to_csv(self.dirpath + 'backup_tid_store.csv',mode='a',header=False,index=False)
+
```
Comment (Aniketh01) [inline twitter_bot.py]: At a code stand point looks good to me!

While you can separate this into a completely new function.

**Comment by Aniketh01** [comment]
Meanwhile, I would like you to design a test case for this PR. That is, collect or create a tied_store with tweets having just 1 relevant keyword. All the elements in the tid_store should be of this type alone. Thus, see if that is being fetched or not.

Even better would be to design a python test. Have a tests/ dir and add your test code there. Use the API from nawab and it's associated bot classes and emulate a simple scenario.

**Review by Aniketh01 — APPROVED** [review]
This looks neat! I'm very well impressed by the tests wrote as well. Well done. LGTM!

Just a minor suggestion, update the folder name test to tests/ :)

**Comment by Aniketh01** [comment]
Also, revamp the commit messages a little bit :)

**Comment by ananthanandanan** [comment]
Ok made the necessary changes as requested. Its ready to merge.

**Pull request merged**